GSA-TTS/identity-handbook
# Improve searchability of redirect pages

## 🛠 Summary of changes

Updates the new search behavior to improve searchability of redirect links.

We have a number of external links as stub articles, and these are currently hard to search for, e.g. "org chart".

Changes include:

- Extend [`jekyll-redirect-from` base redirect template](https://github.com/jekyll/jekyll-redirect-from/blob/master/lib/jekyll-redirect-from/redirect.html) to include annotations to enable search indexing
- And (and require) descriptions for redirect page stubs
- Exclude category pages from indexing

## 📜 Testing Plan

1. Go to preview URL
2. Search for a redirect link, e.g. "org chart"
3. See useful results

## 👀 Screenshots

Before|After
---|---
![image](https://github.com/user-attachments/assets/d4270de7-7d4f-4f3f-b8bd-db73d6e98a90)|![image](https://github.com/user-attachments/assets/c2748244-b6dc-4bf2-b1ad-c2261b8a29b4)
